app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,27 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Assignment Branch Condition size for create is too high. [16.16/15]](https://www.codacy.com/app/conta-srdr/forsythia/file/2130834666/issues/source?bid=3139784&fileBranchId=3147671#l5)